install_config/configuring_openstack.adoc

```diff
@@ -184,3 +171,12 @@ container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
 [[openstack-applying-configuration-changes]]
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[openstack-configuring-zone-labels]]
+=== Configuring Zone Labels for Dynamically Created OpenStack PVs 
+
+Administrators now have the option to configure zone labels for dynamically
+created OpenStack PVs. This option is useful if the OpenStack Cinder zone name
+does not match the compute zone names. For example, if there is only one Cinder
+zone and many compute zones. Administrators can create Cinder volumes
+dynamically and then check the labels.
```

Comment:
I think that you do need to explain how to add the label. I hope you get the info you asked for in the BZ.  :)